tests/storage/snapshot.go

```diff
@@ -733,6 +734,71 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				}
 			})
 
+			It("should report appropriate event when freeze fails", func() {
```

Comment:
@mhenriks this could have been a unit test.

Reply:
Yeah my concern was that there are currently no unit tests in that package